### PR TITLE
honeycomb-server

### PR DESCRIPTION
**Pull request by danielbayerlein**
PR for #75

### CODE REVIEW

**Comment by danielbayerlein** [comment]
For `honeycomb-asset` see #84

File: packages/honeycomb-layout/package.json [hunk 1]
```diff
@@ -10,19 +10,16 @@
     "start": "npm run start:production",
     "start:development": "cross-env NODE_ENV=development nodemon src/index.js --exec babel-node --watch src/ --ext js",
     "start:production": "cross-env NODE_ENV=production node dist/index.js",
-    "test": "npm run lint && npm run test:unit",
```
Comment (chrishelgert) [inline packages/honeycomb-layout/package.json]: sure we don't want any tests for honeycomb-layout?
we won't see anything breaking through pull-request or dependency updates

File: packages/honeycomb-layout/src/index.js [hunk 2]
```diff
@@ -1,30 +1,16 @@
-import Hapi from 'hapi';
-import Hoek from 'hoek';
+import HoneycombServer from 'honeycomb-server';
```
Comment (chrishelgert) [inline packages/honeycomb-layout/src/index.js]: why UpperCase?
i think we should use uppercase only for classes.

File: packages/honeycomb-server/__tests__/lib/index.js [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+      expect(server.register).toHaveBeenCalled();
+    });
+
+    it('should be return a Promise', () => {
+      expect(HoneycombServer.start().constructor.name).toBe('Promise');
+    });
+
+    // eslint-disable-next-line arrow-body-style
+    it('should be return the server', () => {
+      return HoneycombServer.start().then((serverInstance) => {
```
Comment (chrishelgert) [inline packages/honeycomb-server/__tests__/lib/index.js]: .then() should be in the next line i think

File: packages/honeycomb-server/__tests__/lib/index.js [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+        expect(server).toEqual(serverInstance);
+      });
+    });
+
+    // eslint-disable-next-line arrow-body-style
+    it('should have been called console.info()', () => {
+      return HoneycombServer.start().then(() => {
```
Comment (chrishelgert) [inline packages/honeycomb-server/__tests__/lib/index.js]: same here

File: packages/honeycomb-server/__tests__/lib/index.js (hunk 3 above)
Comment (danielbayerlein) [inline packages/honeycomb-server/__tests__/lib/index.js]: b4188371c93c6499b49237bd53b95cc2263487c4

File: packages/honeycomb-server/__tests__/lib/index.js (hunk 4 above)
Comment (danielbayerlein) [inline packages/honeycomb-server/__tests__/lib/index.js]: b4188371c93c6499b49237bd53b95cc2263487c4

File: packages/honeycomb-layout/src/index.js (hunk 2 above)
Comment (danielbayerlein) [inline packages/honeycomb-layout/src/index.js]: You're right! ➡️ f99bd2553dce54177b060bcf48781e6058068310

File: packages/honeycomb-layout/package.json (hunk 1 above)
Comment (danielbayerlein) [inline packages/honeycomb-layout/package.json]: I added a test for `honeycomb-layout` ➡️ 2f037012e0363421cb9c5970b4ff14f85250694e

**Comment by danielbayerlein** [comment]
@chrishelgert All points from the review are done! 🤝

**Pull request merged**